contribs/gmf/apps/desktop/index.html

```diff
@@ -111,6 +111,49 @@
           </gmf-backgroundlayerselector>
         </div>
         <gmf-map class="gmf-map" gmf-map-map="mainCtrl.map"></gmf-map>
+
+        <!--infobar-->
+        <div class="footer" ng-class="{'active': mainCtrl.info}">
+          <button class="btn btn-sm fa map-info ng-cloak" ng-click="mainCtrl.info = !mainCtrl.info"
+                  ng-class="{'fa-angle-double-up': !mainCtrl.info, 'fa-angle-double-down': mainCtrl.info}"></button>
+
+          <div ngeo-scaleselector="mainCtrl.scaleSelectorValues"
+               ngeo-scaleselector-map="mainCtrl.map"
+               ngeo-scaleselector-options="mainCtrl.scaleSelectorOptions"></div>
+          <div id="scaleline"></div>
+          <div class="pull-right">
+            <gmf-mouseposition
+                 gmf-mouseposition-map="mainCtrl.map"
+                 gmf-mouseposition-projections="::mainCtrl.mousePositionProjections">
+            </gmf-mouseposition>
+            <span
+              gmf-elevation
+              gmf-elevation-active="mainCtrl.info"
+              gmf-elevation-elevation="elevationValue"
+              gmf-elevation-layer="mainCtrl.elevationLayer"
+              gmf-elevation-layers="::mainCtrl.elevationLayers"
+              gmf-elevation-map="mainCtrl.map">
+            </span>
```

Comment:
Do you need a specific `<span>`? Isn't it possible to use the directive directly within the div below?